<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drv_kitsClass</class>
 <widget class="QMainWindow" name="drv_kitsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>drv_kits</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_nextstp">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一步</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_disp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(97,195,195);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_center">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>811</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_ftp"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_usrdoc"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_usrdoc">
   <property name="icon">
    <iconset resource="drv_kits.qrc">
     <normaloff>:/drv_kits/pic/wenhao.png</normaloff>:/drv_kits/pic/wenhao.png</iconset>
   </property>
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="drv_kits.qrc">
     <normaloff>:/drv_kits/pic/about.png</normaloff>:/drv_kits/pic/about.png</iconset>
   </property>
   <property name="text">
    <string>关于...</string>
   </property>
  </action>
  <action name="action_ftp">
   <property name="icon">
    <iconset resource="drv_kits.qrc">
     <normaloff>:/drv_kits/pic/ftp.png</normaloff>:/drv_kits/pic/ftp.png</iconset>
   </property>
   <property name="text">
    <string>SO 文件传输</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="drv_kits.qrc"/>
 </resources>
 <connections/>
</ui>
